v19.2/window-functions.md

```diff
@@ -12,23 +12,76 @@ For a complete list of supported window functions, see [Functions and Operators]
 All [aggregate functions][aggregate-functions] can also be used as [window functions][window-functions]. For more information, see the [Examples](#examples) below.
 {{site.data.alerts.end}}
 
-{{site.data.alerts.callout_info}}
-The examples on this page use the `users`, `rides`, and `vehicles` tables from our open-source fictional peer-to-peer vehicle-sharing application, [MovR](movr.html).
-{{site.data.alerts.end}}
+## Window definitions
+
+Window frames are defined in [`OVER` clauses](sql-grammar.html#over_clause) or [`WINDOW` clauses](sql-grammar.html#window_clause).
+
+### Syntax
+
+<div class="filters clearfix">
+  <button style="width: 15%" class="filter-button" data-scope="basic">Basic</button>
+  <button style="width: 15%" class="filter-button" data-scope="expanded">Expanded</button>
+</div>
+
+<br><br>
+
+<div class="filter-content" markdown="1" data-scope="basic">
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/window_definition.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------|-------------
+`window_name` | The name of the new window frame.
+`opt_existing_window_name` | An optional name of an existing window frame, defined in a different window definition.
+`opt_partition_clause`  | An optional `PARTITION BY` clause. See [Define Table Partitions](partitioning.html) for details.
```

Comment:
I don't think the `PARTITION BY` clause we use in window functions is the same as the `PARTITION BY` syntax we introduced for geo-distributed clusters.  The window function `PARTITION BY` syntax is [also supported by PG](https://www.postgresql.org/docs/9.1/tutorial-window.html).

 Pretty sure we overloaded this phrase for the geo-distributed use case.

tl;dr: the link to "Define Table Partitions" can be removed

  @yuzefovich please shout if you disagree with the above just in case I'm misremembering somehow.

Reply:
Yes, you're right - those two are different unrelated things.

Reply:
Ahh, OK. That makes more logical sense. Thanks Rich!